opentelemetry-sdk/src/metrics/aggregators/histogram.rs

```diff
@@ -77,8 +81,17 @@ impl Histogram for HistogramAggregator {
     }
 }
 
+impl Aggregation for HistogramAggregator {
+    fn kind(&self) -> &AggregationKind {
+        &AggregationKind::SUM
```

Comment:
Shouldn't it be `&AggregationKind::HISTOGRAM`?

Reply:
ya good catch